azure-pipelines.yml

```diff
@@ -0,0 +1,170 @@
+# Ruby
+# Package your Ruby project.
+# Add steps that install rails, analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/ruby
+
+name: '$(SourceBranchName)-$(Build.SourceVersion) Build-$(Build.BuildId)'
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - develop
+
+pr:
+  autoCancel: true
+  branches:
+    include:
+      - develop
+      - feature/*
+
+variables:
+  application.name: 'et-ccd-export'
+  azure.subscription.endpoint: 'hmcts-pet'
+  manual.branch.prefix: 'develop'
+
+jobs:
+- job: TestApplication
+
+  pool:
+    vmImage: 'Ubuntu-16.04'
+  
+  steps:
+  - task: UseRubyVersion@0
+    inputs:
+      versionSpec: '>= 2.5'
+
+  - script: |
+      export ENABLE_COVERAGE=true
+      export DB_USERNAME=postgres
+      export DB_PORT=5433
+      export CC_TEST_REPORTER=935a4195bd35a993666acc398a01a53daf4e7f11635e90a51c8f30b44cbe1ec9
+      sudo apt-get update
+      sudo apt-get install libpq-dev mcrypt libmcrypt-dev pdftk
+      chmod +x ./bin/wait-for-it.sh
+      sudo rm /usr/local/bin/docker-compose
+      curl -L https://github.com/docker/compose/releases/download/1.21.0/docker-compose-Linux-x86_64 > docker-compose
+      chmod +x docker-compose
```

Comment:
We dont use docker-compose in any of the tests - there are no support services required ?